client/src/views/SignupCitizen.tsx

```diff
@@ -91,6 +160,7 @@ function SignupCitizen() {
 
   const handleSubmit = async (evt: React.FormEvent) => {
     evt.preventDefault();
+    handleNext();
     setIsLoading(true);
     // Backend doesn't need accept_terms. If a user is signed up they have agreed to the terms
     delete formData.accept_terms;
```

Comment:
while we're here let's change the fetch line to use @esteban-gs 's new endpoints 🥳 
so in [this new account-manager controller](https://github.com/Nonprofit-Exchange-Hub/web-app/blob/f2c3e01df8c171c7ff3d0d57674467f1af073cc5/server/src/acccount-manager/account-manager.controller.ts) it looks like we should be hitting `/users/register`

Reply:
sorry, i think it's `/auth/register` actually
cc: @esteban-gs to confirm

Reply:
YEs!